tests/test_node.py

```diff
@@ -1,36 +1,98 @@
 from PyRDF import *
 import unittest
 
```

Comment:
Not all the tests in this file are documented, e.g. `test_dfs_graph_with_pruning_actions`, `test_dfs_graph_with_pruning_transformations`, ... Please make sure all of them are documented.